inc/sitemaps/class-sitemap-timezone.php

```diff
@@ -81,8 +81,13 @@ private function determine_timezone_string() {
 		// Adjust UTC offset from hours to seconds.
 		$utc_offset *= HOUR_IN_SECONDS;
 
-		// Attempt to guess the timezone string from the UTC offset.
-		$timezone = timezone_name_from_abbr( '', $utc_offset );
+		// Attempt to guess the timezone string from the UTC offset and daylight save mode.
+		$timezone = timezone_name_from_abbr('', $utc_offset, 1 );
```

Comment:
As the documentation in PHP show, the default value of isDST is -1 by default, which takes both daylight saving and not daylight saving into account.

That -should- provide everything we need.
Why would be assume daylight saving as first option? If we are in the winter time this would potentially cause problems with an offset which is not correct.

http://php.net/manual/en/function.timezone-name-from-abbr.php